girder/api/v1/collection.py

```diff
@@ -113,7 +113,9 @@ def getCollectionDetails(self, collection):
         .modelParam('id', model=CollectionModel, level=AccessType.READ)
         .jsonParam('mimeFilter', 'JSON list of MIME types to include.', requireArray=True,
                    required=False)
-        .produces('application/zip')
+        # Tell swagger that we produce binary data; it doesn't treat
+        # application/zip as binary
+        .produces('application/octet-stream')
```

Comment:
I think we should keep this as `application/zip` since that's the correct content-type. Seems like we can fix this by just adding some logic within `produces` itself that converts internally to `application/octet-stream` if the value doesn't match a whitelist. That way, once swagger fixes this bug, we can remove that logic in a central location and everything should work.

Reply:
Yes, that is a better way to do this.